packages/scene-composer/src/components/three-fiber/DataOverlayComponent/DataOverlayDataRow.tsx

```diff
@@ -35,7 +36,7 @@ export const DataOverlayDataRow = ({
       if (!binding.valueDataBinding) {
         return;
       }
-      const values: Record<string, unknown> = dataBindingValuesProvider(
+      const values: Record<string, Primitive> = dataBindingValuesProvider(
```

Comment:
What does Primitive supports?

Reply:
number, string, boolean

Reply:
So we don't want to support the other primitive data types?

Reply:
this change is just to make `unknown` a more restricted type. the actually logic to handle the value is unchanged. I believe we cannot handle other data type in anyway. so this won't change the actual functionality

Reply:
That means we are assuming here type of a value is known at the compilation time when receiving the data from the an external source. Something to watch for if we haven't tested this scenario.